src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -255,6 +261,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       ExpectedPlatformPackages="@(ExpectedPlatformPackages)"
       SatelliteResourceLanguages="$(SatelliteResourceLanguages)"
       DesignTimeBuild="$(DesignTimeBuild)"
+      ContinueOnError="$(_ResolvePackageAssetsContinueOnError)"
```

Comment:
Did I see elsewhere from @davkean that design time builds set $(ContinueOnError)? If so, should we use that?

Reply:
Oh, I see that was #3399. What was the build error? We can work together to validate it if it’s a better pattern.

Reply:
Here is an existing usage: https://github.com/microsoft/msbuild/blob/0cd0e92a7243088977d31b56626b56d6116de016/src/Tasks/Microsoft.Common.CurrentVersion.targets#L2145. It would be good to understand why that isn't work, are we not setting the property correctly in CPS?

Reply:
So if we can get it down to just that, it'd be a lot cleaner. We don't even need the part that default ContinueOnError to false do we? I'd like to push on matching the precedent in common targets.

Reply:
It is already defaulted here: https://github.com/microsoft/msbuild/blob/0cd0e92a7243088977d31b56626b56d6116de016/src/Tasks/Microsoft.Common.CurrentVersion.targets#L608. 

So at this point I would expect #3399, minus the defaulting. If that doesn't work, I'd like to understand why before doing anything more complicated.